examples/solids/index.rst

```diff
@@ -0,0 +1,479 @@
+.. _example-petsc-elasticity:
+
+Solid mechanics elasticity mini-app
+========================================
+
+This example is located in the subdirectory :file:`examples/solids`.
+It solves the steady-state static momentum balance equations using unstructured high-order finite/spectral element spatial discretizations.
+As with the :ref:`example-petsc-navier-stokes` case, the solid mechanics elasticity example has been developed using PETSc, so that the pointwise physics (defined at quadrature points) is separated from the parallelization and meshing concerns.
+
+In this mini-app, we consider three formulations used in solid mechanics applications: linear elasticity, Neo-Hookean hyperelasticity at small strain, and Neo-Hookean hyperelasticity at finite strain.
+We provide the strong and weak forms of static balance of linear momentum in the small strain and finite strain regimes.
+The stress-strain relationship (constitutive law) for each of the material models is provided.
+Due to the nonlinearity of material models in Neo-Hookean hyperelasticity, the Newton linearization of the material models is provided.
+
+.. note::
+
+   Linear elasticity and small-strain hyperelasticity can both by obtained from the finite-strain hyperelastic formulation by linearization of geometric and constitutive nonlinearities.
+   The effect of these linearizations is sketched in the diagram below, where :math:`\bm \sigma` and :math:`\bm \epsilon` are stress and strain, respectively, in the small strain regime, while :math:`\bm S` and :math:`\bm E` are their finite-strain generalizations (second Piola-Kirchoff tensor and Green-Lagrange strain tensor, respectively) defined in the reference configuration, and :math:`\mathsf C` is a linearized constitutive model.
+
+   .. math::
+      \begin{matrix}
+      \text{Finite Strain Hyperelastic} & \underset{\bm S = \mathsf C \bm E}{\overset{\text{constitutive}}{\LARGE \longrightarrow}} & \text{St. Venant-Kirchoff} \\
+      \text{\scriptsize geometric} {\LARGE \ \downarrow\ } \scriptsize{\bm E \to \bm \epsilon} & & \text{\scriptsize geometric} {\LARGE \ \downarrow\ } \scriptsize{\bm E \to \bm \epsilon} \\
+      \text{Small Strain Hyperelastic} & \underset{\bm \sigma = \mathsf C \bm \epsilon}{\overset{\text{constitutive}}{\LARGE \longrightarrow}} & \text{Linear Elastic} \\
+      \end{matrix}
+
+.. _running-elasticity:
+
+Running the mini-app
+----------------------------------------
+
+There are five required command line options and a variety of additional command line options for this mini-app.
+The four required command line options are :code:`-mesh`, :code:`-degree`, :code:`-E`, and :code:`-nu`. Additionally, at least one boundary condition must be set, using :code:`-bc_zero` or :code:`-bc_clamp`.
+
+To set the ExodusII mesh file, use the :code:`-mesh` option and the file path to the mesh file::
+
+   ./elasticity -mesh ./meshes/meshes/cylinder8_672e_4ss_us.exo
```

Comment:
https://github.com/ArashMehraban/CeedPetscSolid/tree/master/meshes
or
https://github.com/jeremylt/ceedSampleMeshes
We have a bunch of meshes here. Would either of these work?

Reply:
To even just run the code with a given mesh a user would need Cubit?

Reply:
Cubit is only required to generate a mesh.

Reply:
Your repository is fine. Better not to mix up access to meshes with the legacy version of the code.

Reply:
That is what I thought. But then I'm not entirely following why not to push at least the meshes mentioned in the sample runs, or as Jed said to make them available for download somewhere and document that, specifying the proper instructions to run the example. Otherwise one would just copy and paste this sample run and it would crash

Reply:
Updated with a link.